src/main/webapp/data/userMapData.json

```diff
@@ -945,6 +966,7 @@
       "University of Moratuwa"
     ],
     "Sweden": [
+    	  "Chalmers University",
```

Comment:
Extra spaces here. Let's keep it align with the lines below

Reply:
Hmm... I suspect this is also the reason that `npm run lint` failed on Travis.

Reply:
I don't see the space problem...did someone already fix this?

Reply:
I'm referring to the spaces/tab before the university name. Can you see from above that this line is not aligned with the other university names above & below?

Reply:
I know, I must've fixed it on my local file before you made note of it. I'll be sure to double check spacing.

Reply:
Hmm... I don't think you have fixed that, since I can still see the extra spaces here. The Travis test won't pass until you've made this change.

Reply:
I found the error you were talking about, but for some reason on my ide the spacing differences were not visible...I deleted and rewrote the Chalmers line and everything seems to be working fine

Reply:
Yea it's passing now

Reply:
You can try opening this file using another IDE/text editor.